app/controllers/associations.js

```diff
@@ -138,34 +124,3 @@ export default class IndexController extends Controller {
     this.sort = '-created-on';
   }
 }
-
-// We store the postalCodes cell in module scope so the previous data is retained between data fetches.
-// This prevents the select from temporarily showing an empty selection.
-const postalCodes = cell([]);
```

Comment:
Removing this wasn't needed for the fix, but this setup was a lot more complicated than needed, so switched to the setup we're using for the other filters.